lib/patch-series.ts

```diff
@@ -624,6 +623,8 @@ export class PatchSeries {
                 throw new Error(`Could not extract cover letter Message-ID`);
             }
 
+            const midTimeStamp = coverMid.match(/cover(\.[0-9]+\.?)/);
+            const midTS = midTimeStamp ? midTimeStamp[1] : ".";
```

Comment:
Those are two `const` variables with very similar names, i.e. the same purpose, and they include the dots (but maybe not a trailing one?).

How about this instead?

```typescript
let timeStamp = coverMid.match(/cover\.([0-9]+\)/);
timeStamp = timeStamp ? timeStamp[1] : `${Date.now()}`;
```

Later on, you would use this as `.${timeStamp}.`.

Reply:
timeStamp in the first assignment is an array and the second use is as a string.  Typescript throws an error - I will keep two vars for the moment (unless you have a better suggestion).

The original code with extra variable midTS was to overcome typescript rejecting midTimeStamp[1] as possibly not defined.  That was kind of the idea - get a runtime (not compile time) error if the regex for midTimeStamp failed.  The match will not fail (the value will always be there) but we have to write guard code for it (assuming git does not change).  Sorry for letting some coding philosophy slip in there.

Reply:
> timeStamp in the first assignment is an array and the second use is as a string. Typescript throws an error - I will keep two vars for the moment (unless you have a better suggestion).

Ah. Yes, I would have the suggestion to specify the type explicitly: `let timeStamp: string[] | string | null` ;-)

But as I stated elsewhere, I would also be fine with this:

```typescript
const match = coverMid.match(/cover\.([0-9]+\)/);
const timeStamp = match ? match[1] : `${Date.now()}`;
```

Reply:
Oh, I missed the update... Nevermind! ;-)